Yafc.Parser/Data/FactorioDataDeserializer.cs

```diff
@@ -21,16 +21,19 @@ internal partial class FactorioDataDeserializer {
         }
 
         result = GetObject<T>(name);
+
         return true;
     }
 
     private T? GetRef<T>(LuaTable table, string key) where T : FactorioObject, new() {
         _ = GetRef<T>(table, key, out var result);
+
         return result;
     }
 
     private Fluid GetFluidFixedTemp(string key, int temperature) {
         var basic = GetObject<Fluid>(key);
```

Comment:
`Fluid`? (Since you're replacing `var` elsewhere too)

Reply:
The `var`s that were replaced were replaced only by Code Cleanup. I did not go after vars in this PR by myself.